.vscode/launch.json

```diff
@@ -15,6 +15,27 @@
                 "nl_program",
                 "--num_train_examples",
                 "1",
+<<<<<<< HEAD
```

Comment:
Haha somehow a merge conflict ended up here. It sounds like you have `commit`-ment issues!

I'm going to add one more commit to remove some of this stuff, and then when we're finished we can squash everything down into a single commit for posterity.